lib/services/slack.rb

```diff
@@ -25,22 +27,32 @@ def receive_audit
         audit.description
       end
       
-    send_message(fallback: "#{user} #{audit.description}",
-      pretext: "#{user} #{link}",
+    send_message(
       username: "Aha!",
       icon_url: "https://secure.aha.io/assets/logos/aha_square_300.png",
-      fields: audit.changes.collect do |change|
-        {
-          title: change.field_name,
-          value: html_to_plain(change.value),
-          short: false
-        }
-      end)
+      attachments: [
+        fallback: "#{user} #{audit.description}",
+        pretext: "*#{user}* #{link}",
+        color: "good",
```

Comment:
I am not sure we want to set a color here. This puts a green bar next to each item. Without this setting it was light gray.

Here is an example showing the before and after.
![screen shot 2015-10-01 at 3 32 13 pm](https://cloud.githubusercontent.com/assets/161640/10235480/b22ad994-6851-11e5-8c83-97410ab2f71a.png)